src/components/Label/Label.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import FormLabel from 'react-bootstrap/FormLabel'
+
+interface Props {
```

Comment:
I think we might want to support `htmlFor` so we can tie form controls to their labels if necessary.

What do do you think?

Reply:
Happy to add this prop, on that note what are you thoughts on adding a column prop to support horizontal forms? Do we have any plans to use horizontal forms or any plans to in the future?

Reply:
No plans that I no of yet. But if it’s simple enough to add, no harm in it!